source/reference/color_schemes.rst

```diff
@@ -0,0 +1,256 @@
+   is set to ``true``.
+
+``bracketsForeground``
+   Background color of the brackets
+   when the caret is next to a bracket.
+   Only used when the ``match_brackets`` setting
+   is set to ``true``.
+
+``bracketsOptions``
+   XXX
+
+   Options: ``underline``, ``stippled_underline``, ``squiggly_underline``.
+   ``underline`` indicates the text should be drawn
+   using the given color, not just the underline.
+
+``caret``
+   Color of the caret.
+
+``findHighlight``
+   Background color of regions matching the current search.
+
+``findHighlightForeground``
+   Background color of regions matching the current search.
+
+``foreground``
+   Foreground color for the view.
+
+``guide``
+   Color of the guides displayed to mark nesting levels.
+
+``gutter``
+   Background color of the gutter.
+
+``gutterForeground``
+   Foreground color of the gutter.
+
+``inactiveSelection``
+   Color of inactive selections (inactive view).
+
+``invisibles``
+  Ignored by Sublime Text.
+
+``lineHihglight``
+   Color of the line the caret is in.
+   Only used when the ``higlight_line`` setting is set to ``true``.
+
+``selection``
+   Color of the selection regions.
+
+``selectionBackground``
+   Background color of the selection regions.
+
+``selectionBorder``
+   Color of the selection regions’ border.
+
+``shadow``
+   Color of the shadow effect when the buffer is scrolled.
+
+``shadowWidth``
+   Width ot the shadow effect when the buffer is scrolled.
+
+``stackGuide``
+   XXX
+   Only used if the ``indent_guide_options`` setting
+   is set to ``draw_active``.
+
+``tagsForeground``
+   Color of tags when the caret is next to a tag.
+   Only used when the ``match_brackets`` setting
+   is set to ``true``.
+
+``tagsOptions``
+   XXX when the caret is next to a tag.
+   Only used when the ``match_brackets`` setting
+   is set to ``true``.
+
+   Options: ``underline``, ``stippled_underline``, ``squiggly_underline``.
+   ``underline`` indicates the text should be drawn
+   using the given color,
+   not just the underline.
+
+``highlight``
+   Background color for reggions added via ``sublime.add_regions()``
+   with the ``sublime.DRAW_OUTLINED`` flag added.
+
+``highlightForeground``
+   Foreground color for reggions added via ``sublime.add_regions()``
+   with the ``sublime.DRAW_OUTLINED`` flag added.
+
+
+Scoped Settings
+---------------
+
+Associated with a particular scope.
+
+
+``name``
+   Descriptive name of the item.
+
+``scope``
+   Target scope name.
+
+``settings``
+   Container for settings.
+
+   Valid settings are:
+
+``fontStyle``
+   Style of the font.
+
+   Options: ``bold``, ``italic``.
+
+``foreground``
+   Foreground color.
+
+``background``
+   Background color.
+
+
+Sublime Text Settings Related to Color Schemes
+==============================================
+
+``color_scheme``
+   Path to a color scheme file
+   relative to the Data folder
+   (example: Packages/User/Color Schemes - Custom/Fictitious.tmTheme).
```

Comment:
How about refering to a default color scheme? Also, I suggest using `:file:` or just ``` `` ```.